gpslogger/src/main/java/com/mendhak/gpslogger/ui/fragments/settings/CustomUrlFragment.java

```diff
@@ -270,6 +271,7 @@ public boolean onResult(@NonNull String dialogTag, int which, @NonNull Bundle ex
 
         if(dialogTag.equalsIgnoreCase(PreferenceNames.LOG_TO_URL_PATH)){
             String url = extras.getString(PreferenceNames.LOG_TO_URL_PATH);
+            url = url.replaceAll("\n","");
```

Comment:
These two changes should make working with Custom URL easier.  The input text is larger now.  Problem - someone might add a newline! So I just remove it here.